---
{"dg-publish":true,"permalink":"/medical-school/year-1/funmed/week-2/dna-structure-and-replication/","tags":["funmed"]}
---

```table-of-contents
```
# Learning Objectives
- [ ] Describe, using simple diagrams, the structure of DNA and its organisation into nucleosomes, chromatin, and chromosomes
- [ ] Outline the mechanism of DNA synthesis (replication) and describe how some antibiotics interfere in this process
- [ ] Explain the very low level of mistakes in the DNA replication process
- [ ] Outline methods of DNA repair, with examples of clinical consequences of defective repair mechanisms
# Types of Bases
##  Pyrimidines
- Cystosine
- Thymine
- Uracil
- Six-membered, single-ringed structures

You can remember these as the ones with a Y in them.
## Purines
- Adenine
- Guanine
- Nine-membered, double-ringed structures

P**u**rine = H**u**ge
P**y**rimidines = Tin**y**

AT = 2 H-bonds
CG = 3 H-bonds

# DNA shape/size
- double helix
- major grooves of 2.2nm
- minor grooves of 1.2nm
- length of turn 3.4nm
- diameter 2nm

# Compaction of DNA
DNA condensed into nucleosome beads (8 histones, 146 DNA base pairs)
Nucleosome beads condensed into a **solenoid**
- association of H1 histone proteins with each nucleosome to maintain structure
# DNA Replication
##  Leading Strand
- **Gyrase*** (a topoisomerase) unwinds DNA
- **Helicase** breaks H-bonds
- activated free nucleotides line up by complementary base pairing
- **Primase** binds to start of sequence to indicate where **DNA polymerase** should bind
- **DNA polymerase** catalyses formation of phosphodiester bonds in 5' to 3'
## Lagging strand
- **DNA primase** and **RNA primers**
- **DNA polymerase** catalyses formation of phosphodiester bonds in 5' to 3'
- **DNA ligase** "fills in the gaps" between Okazaki fragments

eukaryotic cells have thousands of origins of replication
prokaryotic cells only have one

## Interference of DNA replication using antibiotics
- topoisomerases prevent DNA supercoiling
	- quinolones target topoisomerases to cause supercoiling in Gram negative bacteria
		- gyrase and topisomerase IV converted into toxic enzymes which fragment the bacterial chromosome

# Proofreading and Editing DNA
## DNA mismatch repair
- errors in new DNA strand cause a "bump" in the DNA
- MutS binds to mismatched base pair
- MutL scans nearby DNA for nick
	- nick = single strand break which provides signals for the mismatch system to fix up
	- then triggers strand removal to the mismatch
## Single strand repairs
if not repaired, these lead to mutations
### Base Excision Repair
- correction of single bases
### Nucleotide Excision Repair
- 


## Double strand repairs
if not repaired, these lead to genetic instability

### Non-homologous end joining
- the two broken ends of the chromosome are simply glued back together
	- this repair mechanism is “messy” and typically involves the loss, or sometimes addition, of a few nucleotides at the cut site
		- tends to produce a mutation, but this is better than the alternative (loss of an entire chromosome arm).
### Homologous recombination
- information from the homologous chromosome that matches the damaged one (or from a sister chromatid, if the DNA has been copied) is used to repair the break
	- two homologous chromosomes come together, and the undamaged region of the homologue or chromatid is used as a template to replace the damaged region of the broken chromosome
		- “cleaner” than non-homologous end joining and does not usually cause mutations.